src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonObjectCreationHandlingAttribute.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json.Serialization;
+
+/// <summary>
+/// When placed on a field or property, indicates if member will replaced or populated.
+/// When placed on a type with <see cref="JsonObjectCreationHandling.Populate"/> indicates that all members which are capable of being populated will be.
+/// </summary>
+/// <remarks>
+/// For attributes placed on fields and properties with default resolvers this will be reflected with <see cref="JsonPropertyInfo.ObjectCreationHandling"/>.
```

Comment:
Nit: replace "reflected with" with "mapped to".

Reply:
will be included in the next iteration